interceptor_test.go

```diff
@@ -659,6 +662,29 @@ func TestStreamingMetricsFail(t *testing.T) {
 							Temporality: metricdata.CumulativeTemporality,
 						},
 					},
+					{
+						Name: rpcServerResponsesPerRPC,
+						Unit: unitDimensionless,
+						Data: metricdata.Histogram[int64]{
+							DataPoints: []metricdata.HistogramDataPoint[int64]{
+								{
+									Attributes: attribute.NewSet(
+										semconv.NetPeerNameKey.String(host),
+										semconv.NetPeerPortKey.Int(port),
+										semconv.RPCSystemKey.String(bufConnect),
+										semconv.RPCServiceKey.String(pingv1connect.PingServiceName),
+										semconv.RPCMethodKey.String(PingStreamMethod),
+										attribute.Key(rpcBufConnectStatusCode).String("data_loss"),
+									),
+									Count: 1,
+									Sum:   0,
```

Comment:
How should the handler under error respond? Kept the current behaviour, no response count is reported as no message is sent (only error).